packages/react-devtools-shared/src/devtools/store.js

```diff
@@ -170,13 +170,15 @@ export default class Store extends EventEmitter<{
   // Renderer ID is needed to support inspection fiber props, state, and hooks.
   _rootIDToRendererID: Map<number, number> = new Map();
 
-  // These options may be initially set by a confiugraiton option when constructing the Store.
+  // These options may be initially set by a configuration option when constructing the Store.
   _supportsNativeInspection: boolean = true;
   _supportsProfiling: boolean = false;
   _supportsReloadAndProfile: boolean = false;
   _supportsTimeline: boolean = false;
   _supportsTraceUpdates: boolean = false;
 
+  _supportsReloadAndProfileOverride: boolean = false; // React Native sets this option if it supports reload and profile
```

Comment:
Maybe we could rename this to `supportsReactNativeReloadAndProfile`?

Reply:
I like that, much better name!